frontends/election-manager/src/screens/write_ins_transcription_screen.tsx

```diff
@@ -150,9 +155,22 @@ export function WriteInsTranscriptionScreen({
             <p>overflow.</p>
           </TranscriptionMainContentContainer>
           <TranscriptionPaginationContainer>
-            <Button onPress={onClickPrevious}>Previous</Button>
-            <Text bold>2 of 242</Text>
-            <Button onPress={onClickNext}>Next</Button>
+            <Button
+              disabled={!onClickPrevious}
+              onPress={onClickPrevious || (() => null)}
+            >
+              Previous
+            </Button>
+            <Text bold>
+              {ballotIdxBeingAdjudicated + 1} of{' '}
+              {ballotsBeingAdjudicated.length}
+            </Text>
+            <Button
+              disabled={!onClickNext}
+              onPress={onClickNext || (() => null)}
```

Comment:
Maybe make these default values on props?

```ts
function noop() {
  // nothing to do
}

export function WriteInsTranscriptionScreen({
   contest,
   election,
   ballotIdxBeingAdjudicated,
   ballotsBeingAdjudicated,
   onClickNext = noop,
   onClickPrevious = noop,
   onClose,
```

Reply:
If we set `noop` as the default, we'd need to change the `disabled` attribute on those buttons to `disabled={onClickNext.name === 'noop'}` which I don't love, but I do like the idea of pulling `noop` out into a function, so I'll do that.